src/components/Messages/Messages.jsx

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from "react-redux";
+import { set } from "../../slices/MessagesSlice";
+import store from "../../store";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import "./messages.scss";
+import "./ConsoleInterceptor.js";
+
+// A component that displays error messages
+function Messages() {
+  const messages = useSelector(state => state.messages);
+  return (
+    <div className="messages-pane">
+      {messages.items.map((message, index) => {
+        return (
+          <Alert icon={false} key={index} severity={message.severity}>
+            <AlertTitle>{message.title}</AlertTitle>
+            {message.text}
+          </Alert>
+        );
+      })}
+    </div>
+  );
+  return res;
+}
+// Deletes expired message (should be in slice file but I cannot find a way to access the store from there)
+const MESSAGES_DISPLAY_DURATION = 5000;
+window.setInterval(() => {
```

Comment:
More i think about it, do we need to remove old ones? Why not keep it an alert so users have to make a conscious deciion that they've read it.

Reply:
I think it depends on the number of messages that are popping up. For the time being this number should be low so I agree.

Reply:
The user has a button to close the alert now. But I still close the alert automatically after 60s just in case.